### DISCUSSION TITLE
Strings without a label

### DISCUSSION THREAD

**Discussion by ghubz** [WeblateOrg/weblate#7111] [Q&A]
Hi,

I've searched the documentation but cannot seem to understand how this works. In a component listing under a language I see "Strings without a label XXXX" 

To my understanding this is unrelated to the "Labels" functionality because I haven't created any labels however on some components this is not shown at all.

So what does that actually mean if you don't have any labels and why is it shown on some and others not? I've tried replicating by creating a new project/component using automatic translations, marking strings as need editing etc but couldn't. 

Thanks

**Comment by nijel** [comment]
Some labels are automatically created by Weblate, see https://docs.weblate.org/en/latest/user/translating.html#labels

**Reply by ghubz** [reply]
I already read that but it isn't really clear. 

Is the following correct?: 
I have 20 strings. I automatically translate 6 of them and another 6 are marked as needed review. The remaining 8 will be marked as  "Strings without a label". If I manually translate any of them without automation or marking as editing it will not show "Strings without a label" at all (considering that no other custom labels are used)

is that the case?

**Reply by nijel** [reply]
The "Strings without a label" filter is shown once there are some labels defined for the project. Weblate automatically creates some labels (see above). The link is then shown even in the case these labels are no longer used.
